src/classes/StateTriggerHelper.cls

```diff
@@ -0,0 +1,134 @@
+/**
+    Description     :   State Trigger Helper Class for State_Trigger.
+                        Updating total_district, total_states and Json Data on Country
+                        when inserting, updating or deleting State.
+                        Filling map for JSON.
+    Created BY      :   Aryan Sharma
+    Created Date    :   08-08-2022
+    Revision Logs   :   V_1.0 - Created
+**/
+public class StateTriggerHelper {
+    public static Map<String, Map<String, List<String>>> mapCountryAndStateWithDistricts = 
+        new Map<String, Map<String, List<String>>>(); 
+    //These are the Wrapper Class for JSON
+    public class Country{
+        public String countryName{get; set;}
+        public Integer Total_Number_Of_Districts{get; set;}
+        public Integer Total_Number_Of_States{get; set;}
+        public List<States> states{get; set;} 
+    }
+    public class States{
+        public List<Districts> districts{get; set;}
+        public Integer Total_Number_Of_Districts{get; set;}
+        public String stateName{get; set;}
+    }
+    public class Districts {
+        public String districtName{get; set;}
+    }
+    public static List<Country__c> countryListStateCountAndJSONToUpdate = new List<Country__c>();
+    // Polulating  Country Records when Trigger is on State Object
+    public static void populateCountryRecords(List<State__c> newStateList, Map<Id, State__c> oldStateMap) {
+        Set<Id> countryId = new Set<Id>();
+        Set<Id> newCountryId = new Set<Id>();
```

Comment:
remove the variables if they are not in use and follow the correct naming convention